book/06-github/sections/3-maintaining.asc

```diff
@@ -218,160 +218,160 @@ Mayroon ding ref na `refs/pull/#/merge` sa panig ng GitHub , na kumakatawan sa c
+X-GitHub-Tumatanggap: tonychacon
+ID-ng-Listahan: tonychacon/fade <fade.tonychacon.github.com>
+Listahan-ng-Archive: https://github.com/tonychacon/fade
+Listahan-ng-Paskil: <mailto:reply+i-4XXX@reply.github.com>
+Listahan-ng-Unsubscribe: <mailto:unsub+i-XXX@reply.github.com>,...
+X-GitHub-Address-ng-Tumatanggap: tchacon@example.com
 ----
 
-There are a couple of interesting things here.
-If you want to highlight or re-route emails to this particular project or even Pull Request, the information in `Message-ID` gives you all the data in `<user>/<project>/<type>/<id>` format.
-If this were an issue, for example, the `<type>` field would have been ``issues'' rather than ``pull''.
+Mayroong ilang mga kawili-wiling bagay dito.
+Kung gusto mong i-highlight o i-rutang muli ang mga email sa partikular na proyektong ito o kahit ang Kahilingan na Pull, ang impormasyon sa `ID-ng-Mensahe` ay nagbibigay sa iyo ng lahat ng datos sa format na `<user>/<project>/<type>/<id>`.
+Kung ito isang isyu, bilang halimbawa, ang patlang na `<uri>` sana ay naging mga ``isyu'' sa halip na ``pull''.
 
-The `List-Post` and `List-Unsubscribe` fields mean that if you have a mail client that understands those, you can easily post to the list or ``Unsubscribe'' from the thread.
-That would be essentially the same as clicking the ``mute'' button on the web version of the notification or ``Unsubscribe'' on the Issue or Pull Request page itself.
+Ang mga patlang na `Listahan-ng-Paskil` at `Listahan-ng-Unsubscribe` ay nangangahulugan na kung mayroon kang kliyente ng mail na umiintindi sa mga iyon, madali mong mapaskil sa listahan o ``Unsubscribe'' mula sa thread.
+Iyon ay mahalagang kapareho a pag-click sa pindutan na ``mute'' sa bersyon sa web ng abiso o ``Unsubscribe'' sa pahina mismo ng Isyu o Kahilingan na Pull.
 
-It's also worth noting that if you have both email and web notifications enabled and you read the email version of the notification, the web version will be marked as read as well if you have images allowed in your mail client.
+Makabuluhan ding tandaan na kung pareho kang mayroong mga abiso sa email at web na pinagana at nabasa mo ang bersyon sa email ng abiso, ang bersyon sa web ay mamarkahan bilang nabasa pati na rin kung ikaw ay mayroon mga larawan na pinayagan sa iyong kliyente ng mail.
 
-==== Special Files
+==== Mga Espesyal na File
 
-There are a couple of special files that GitHub will notice if they are present in your repository.
+Mayroong ilang mga espesyal na file na mapapansin ng GitHub kung naroon sila sa iyong repositoryo.
 
 ==== README
 
-The first is the `README` file, which can be of nearly any format that GitHub recognizes as prose.
-For example, it could be `README`, `README.md`, `README.asciidoc`, etc.
-If GitHub sees a README file in your source, it will render it on the landing page of the project.
+Ang una ay ang file na `README`, na maaaring maging halos sa anumang format na nakikilala ng GitHub bilang prosa.
```

Comment:
change prosa to tuluyan

Reply:
hi sir @robin-ho, prose can also be translated to `prosa`.
references: 
* https://translate.google.com.ph/#en/tl/prose
* https://glosbe.com/en/tl/prose